model_compression_toolkit/data_generation/README.md

```diff
@@ -0,0 +1,91 @@
+- *image_granularity*: The granularity of the images for optimization.
+- *image_pipeline_type*: The type of image pipeline to use.
+- *image_normalization_type*: The type of image normalization to use.
+- *image_padding*: Padding to be applied to the images during data generation.
+- *bn_layer_types*: List of BatchNorm layer types to be considered for data generation.
+- *clip_images*: Whether to clip images during optimization.
+- *reflection*: Whether to use reflection during optimization.
+
+## Results Using Generated Data
+### Experamental setup
+##### Quantization Algorithms
+The evaluation of the generated data involved using four quantization algorithms:
+
+- **Regular Post Training Quantization (PTQ)** with 8 bit weights and activations.
+- **Mixed Presicion** (MP) with a compression factor of x8
+- **PTQ using a hardware-friendly Look-Up Table** for 4 bit weights, 8 bit activations.
+- **Enhanced Post Training Quantization (EPTQ)** using 4 bit weights, 8 bit activations.
+
+All setups where tested with symmetric weights and uniform activations.
+
+To ensure reliable results, all experiments were averaged over 5 different random seeds (0-4). 
+
+##### Data Generations Parameters
+The evaluation was performed on the following neural network models:
+
+- Resnet18 and Mobilenet v2 from the torchvision library.
+- Yolo-n from ultralitics.
+ 
+The default data generation configuration was used with 500 iterations for data generation (better results may be achieved with a larger iteration budget).
+- For Resnet18 and Mobilenet v2, 1024 images were generated using a data generation batch size of 32 or 128 and a resolution of 224x224. 
+- For Yolo-n, due to memory limitations, a batch size of 4 was used, and only 128 images were generated with a resolution of 640x640.
+
+Please note that the choice of quantization algorithms and data generation parameters can have a significant impact on the results. The experimental setup provides a foundation for comparing the performance of different models and quantization techniques using the generated data.
+
+|                Model                 |  Resnet18 (69.86)  |Resnet18 (69.86)     | Resnet18 (69.86)    | Resnet18 (69.86) |  Mobilenet v2 (71.89)  | Mobilenet v2 (71.89)  | Mobilenet v2 (71.89) | Mobilenet v2 (71.89) |  Yolo-v8-n (37.26)  |  Yolo-v8-n (37.26)   | Yolo-v8-n (37.26) |
+|:------------------------------------:|:------------------:|:-------------------:|:-------------------:|:----------------:|:----------------------:|:---------------------:|:--------------------:|:--------------------:|:-------------------:|:--------------------:|:-----------------:|
+|  Data type \ Quantization algorithm  |      PTQ W8A8      | MP compression x 8  |    PTQ LUT W4A8     |    EPTQ W4A8     |        PTQ W8A8        | MP compression x 8    |  PTQ LUT W4A8        |      EPTQ W4A8       |      PTQ W8A8       |     PTQ LUT W4A8     |     EPTQ W4A8     |
```

Comment:
Took me a while to get it.
Is it possible to give a different color for this (and the "Model") cell so it would be clear that this is part of the title?
Also, in my mind I'm expecting the first term to refer to the columns, not the rows (I don't know if there is a convention for this, but maybe this is what got me confused).